crates/sui-config/tests/snapshot_tests.rs

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+// To review and fix test failures in this file,
+// 0. Install cargo-insta
+// 1. Run `cargo insta test --review` under `./sui-config`.
+// 2. Review, accept or reject changes.
```

Comment:
I would expand this comment to give a bit more context about this, and list:
- that if you broke one of those tests, you probably have a real impact on a production configuration file, you should call this out, and mark your PR as a breaking change.
- if you are managing a production configuration file, you should add yourself to the CODEOWNERS of the corresponding snapshot tests.
- if you are changing one of those snapshots, you should wait for a review from the CODEOWNER (if any) before merging.

Reply:
Reworded here.